tools/run_tests/python_tests.json

```diff
@@ -13,6 +13,8 @@
   "grpc.framework.face.event_invocation_synchronous_event_service_test",
   "grpc.framework.face.future_invocation_asynchronous_event_service_test",
   "grpc.framework.foundation._later_test",
-  "grpc.framework.foundation._logging_pool_test"
+  "grpc.framework.foundation._logging_pool_test",
```

Comment:
I've resolved the collisions by recreating my commits on top of the current head-of-master. It wasn't any significant trouble. :-)